config/src/config.rs

```diff
@@ -149,6 +149,11 @@ pub struct IndexerConfig {
 #[cfg_attr(test, derive(PartialEq))]
 pub struct DatabaseConfig {
     pub postgres_url: Url,
+    pub postgres_host: String,
+    pub postgres_port: i32,
+    pub postgres_user: String,
+    pub postgres_password: String,
+    pub postgress_db: String,
```

Comment:
You should turn this into an Enum with two variants: `Url { postgres_url: Url }` and `Splitten { .. }`. Also, password and port should be optional.